projects/go-lib/src/lib/components/go-accordion/go-accordion.component.ts

```diff
@@ -74,10 +75,12 @@ export class GoAccordionComponent implements OnInit, AfterContentInit {
 
   private openPanel(panel: GoAccordionPanelComponent): void {
     panel.expanded = true;
+    panel.updateClasses();
   }
 
   private closePanel(panel: GoAccordionPanelComponent): void {
     panel.expanded = false;
```

Comment:
What if we added a setter to the expanded property that then updated the classes as well? This would move the responsibility of resetting the classes back to the child component.